[PATCH] fix build with C++ support and gcc >= 4.4

EOF is defined in stdio, so ensure it is included.

Based on similar patch by Markus Rothe:
http://comments.gmane.org/gmane.comp.web.fastcgi.devel/3205

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 libfcgi/fcgio.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: fcgi-2.4.0/libfcgi/fcgio.cpp
===================================================================
--- fcgi-2.4.0.orig/libfcgi/fcgio.cpp
+++ fcgi-2.4.0/libfcgi/fcgio.cpp
@@ -22,6 +22,7 @@
 #define DLLAPI  __declspec(dllexport)
 #endif
 
+#include <cstdio>
 #include <limits.h>
 #include "fcgio.h"
 
